frontend/app/src/components/history/ClosedTrades.vue

```diff
@@ -155,7 +156,10 @@ const {
   state: trades,
   execute
 } = useAsyncState<Collection<TradeEntry>>(
-  args => fetchTrades(args),
+  async args => {
+    if (!args) return defaultCollectionState<TradeEntry>();
```

Comment:
do we still have a case where args is empty? do you know when this happens?

Reply:
so for the first call, the args is always `undefined`.
I console something inside the `fetchData` but it's not called first.
So I think the useAsyncState, that initiate the first call.

Reply:
https://github.com/vueuse/vueuse/blob/941392cd60c187e757bdf3f1b7445f072b96db0e/packages/core/useAsyncState/index.ts#L29

Ok it seems that by default execute is immediate and it happens automatically.
But instead of doing this we can make it `immediate: false` and call it manually

Reply:
In general, it is always good to check the documentation, and if that is not listing everything also the source code.